public/registration.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Registration</title>
+  body {
```

Comment:
What are other ways to style your code. For example: SCSS.

What are differences?